dts/bindings/spi/gd,gd32-spi.yaml

```diff
@@ -24,3 +24,23 @@ properties:
 
   pinctrl-names:
     required: true
+
+  tx-dma-priority:
```

Comment:
Why isn't this using a `dmas` property like dts/bindings/spi/atmel,sam0-spi.yaml

Reply:
I will apply your pointing out.
dma-gd32.c need fix prior than spi,
so I create new PR #48756.
Could you please review also it?